integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -279,16 +279,22 @@ public int getattr(String path, FileStat stat) {
       stat.st_mtim.tv_sec.set(ctime_sec);
       stat.st_mtim.tv_nsec.set(ctime_nsec);
 
-      // for shell-based group mapping, use the uid and gid of the corresponding user registered in
-      // unix; otherwise use uid and gid of the user running alluxio-fuse
+      // for shell-based group mapping, try to get the uid and gid of file/dir owner.
+      // If the user does not registered in the unix or we use other group mapping service,
+      // uid and gid of the user running alluxio-fuse will be set.
+      long uid = -1;
+      long gid = -1;
       if (mIsShellGroupMapping) {
         String owner = status.getOwner();
-        stat.st_uid.set(AlluxioFuseUtils.getUid(owner));
-        stat.st_gid.set(AlluxioFuseUtils.getGid(owner));
-      } else {
-        stat.st_uid.set(UID);
-        stat.st_gid.set(GID);
+        uid = AlluxioFuseUtils.getUid(owner);
+        gid = AlluxioFuseUtils.getGid(owner);
```

Comment:
what happened previously when we set the user/group to `-1`?

Reply:
It will show as `user:nobody group:nogroup` in the `ls` command
```
root@default:/opt/au# ls -lhat /mnt/alluxio/gcs/my_file.zip
---------- 0 nobody nogroup 107M Oct 13 21:30 /mnt/alluxio/gcs/my_file.zip
```